components/Button.tsx

```diff
@@ -0,0 +1,54 @@
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+export const Button = ({
```

Comment:
this seems to be used not as a button, but as a link. can we either rename to Link or make it a button tag?